x/gov/keeper/deposit.go

```diff
@@ -182,3 +196,16 @@ func (keeper Keeper) RefundDeposits(ctx sdk.Context, proposalID uint64) {
 		return false
 	})
 }
+
+func (keeper Keeper) SupportEGFProposalTotDepositProposal(ctx sdk.Context, first bool, claimCoin sdk.Coins) sdk.Coins {
```

Comment:
Check the spelling of method names SupportEGFProposal Tot DepositProposal  Tot -> Total？